app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/CircleAnnotationActivity.kt

```diff
@@ -69,7 +69,7 @@ class CircleAnnotationActivity : AppCompatActivity() {
 
         // random add circles across the globe
         val circleAnnotationOptionsList: MutableList<CircleAnnotationOptions> = ArrayList()
-        for (i in 0..2000) {
+        for (i in 0..20) {
```

Comment:
curious why did we decrease number here?

Reply:
Other examples are all 20, 2000 is introduced to test drag performance, now we don't need it anymore.